### DISCUSSION TITLE
Support for `byte` and `short` property autoconfiguration?

### DISCUSSION THREAD

**Discussion by alcarraz** [jpos/jPOS#565] [Ideas]
What do you think about implementing `@Config` for byte fields?

We could add a:
```java
if (c.isAssignableFrom(byte.class) || ...)
    field.set(obj, (byte) 0xff & cfg.getInt(config.value())); 
```

And maybe something similar, but would involve a for loop, for a byte array property.

Or we could add a default implementation for `Configuration.getByte()`. And the same for `short`.

to:

https://github.com/jpos/jPOS/blob/f0471696043559b51e3b075d22a6944b932df9f8/jpos/src/main/java/org/jpos/q2/QFactory.java#L458-L478

My motivation is just to lower the boilerplate to configure pinblock formats in a participant. Maybe the byte array support is overkilling, but the plain `byte` support seems to worth the effort. 

The short array may be more reasonable to support easy configuration of MiniGL, layers for example and since we are at it, we can implement all of them.

I'm willing to implement the PR if agreed.

**Comment by ar** [comment]
Wouldn't hurt, but I don't think of a case where we use 'byte' as configuration. `byte[]` is probably more useful (assuming a conversion using `hex2byte`).
